readme.rst

```diff
@@ -1,17 +1,26 @@
 .. image:: documentation/diagrams/logo_navitia_horizontal_fd_gris_250px.png
     :alt: navitia
-    :align: center 
+    :align: center
 
+=========
+ Navitia
+=========
 ``(pronounce [navi-sia])``
 
-.. image::  https://ci.navitia.io/buildStatus/icon?job=navitia_release
-    :alt: build status
+.. class:: no-web no-pdf
+
+    Release branch: |last_release_build|
+
+.. |last_release_build| image:: https://ci.navitia.io/job/navitia_release/badge/icon
```

Comment:
It seems the image can not appear ? Although it's not a regression (maybe some FF rules on ci.navitia.io)